lib/fluent/plugin/in_tail.rb

```diff
@@ -1097,6 +1097,9 @@ def handle_notify
                   end
                 rescue EOFError
                   @eof = true
+                rescue Errno::ESTALE
+                  @log.info "remote file was deleted during read", path: path
```

Comment:
```suggestion
                  @log.info "remote file was deleted during read", path: @path
```
A local variable or method with the name `path` doesn't seem exist.